connection/connection.ts

```diff
@@ -365,7 +373,7 @@ export class Connection {
             if (!tls_enforced) {
               console.error(
                 bold(yellow("TLS connection failed with message: ")) +
-                  e.message +
+                  (e as Error).message +
```

Comment:
Not a fan of type assertions, would prefer if type checks were used.

Reply:
fixed